parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -187,6 +187,58 @@ write_files:
     {{WrapAsVariable "systemdBPFMount"}}
 {{end}}
 
+{{if NeedsContainerd}}
+- path: /etc/containerd/config.toml
+  permissions: "0644"
+  owner: root
+  content: |
+    subreaper = false
+    oom_score = 0
+    [plugins.cri]
+    sandbox_image = "{{GetPodInfraContainerSpec}}"
+    [plugins.cri.containerd.untrusted_workload_runtime]
+    runtime_type = "io.containerd.runtime.v1.linux"
+{{if IsKataContainerRuntime }}
```

Comment:
this looks like dead code, is it possible to specify both `containerd` and `kata-containers`?

Reply:
```go
// NeedsContainerd returns whether or not we need the containerd runtime configuration
// E.g., kata configuration requires containerd config
func (k *KubernetesConfig) NeedsContainerd() bool {
	return k.ContainerRuntime == KataContainers || k.ContainerRuntime == Containerd
}
```

Superfluous, perhaps, but I don't think it's dead.

Reply:
I see, thx